flake.nix

```diff
@@ -0,0 +1,10 @@
+{ inputs.utils.url = "github:numtide/flake-utils";
```

Comment:
`flake-utils` is not needed for what this project is doing.

Reply:
I mean, `flake-utils` is never *needed*, it's just a nicer api. Do you have something more specific to say about it?

Reply:
https://github.com/NixOS/nixpkgs/blob/master/flake.nix#L14-L16

Nicer API? It's actually more lines of code to have `inputs` and now you have an additional dependency.

Reply:
I believe flake-utils eachDefaultSystem is already well know in nix ecosystem, +1 to add it

Reply:
I believe `eachDefaultSystem` is well know in nix ecosystem already. About not being needed, is not easy-purescript-nix creating nix packages?